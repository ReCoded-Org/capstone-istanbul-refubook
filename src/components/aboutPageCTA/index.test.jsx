import React from 'react';
import AboutPageCTA from './';
import renderer from 'react-test-renderer';

it('AboutPageCTA renders correctly', () => {
  const tree = renderer.create(<AboutPageCTA />).toJSON();
  expect(tree).toMatchSnapshot();
});
