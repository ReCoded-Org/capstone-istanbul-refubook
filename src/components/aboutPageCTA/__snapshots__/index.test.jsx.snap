// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPageCTA renders correctly 1`] = `
<div
  className="text-center mx-8 xl:pt-56  sm:pt-64  pt-20 space-y-6 "
>
  <svg
    className="mx-auto"
  >
    imageIcon.svg
  </svg>
  <h2
    className="text-blue-700 text-2xl"
  >
    about.aboutPageCTA.title
  </h2>
  <p
    className="mx-auto lg:px-40 sm:px-20"
  >
    about.aboutPageCTA.text
  </p>
  <button
    className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded-full"
  >
    about.aboutPageCTA.SignUp
  </button>
</div>
`;
